src/casp/ml/services/inference/pca/constants.py

```diff
@@ -0,0 +1,10 @@
+import json
+import os
+import typing as t
+
+import dotenv
+
+dotenv.load_dotenv(dotenv_path="casp/ml/services/inference/.env")
```

Comment:
it feels odd to have two ways of doing this, some via constants some via env vars?

Reply:
I don't think so, as in `.env` I store secret variables like service account credentials. I parse dot env and use it as a secret environment variable. Later on `.env` is supposed to be taken from secret storage and generated by CI. I am happy to see your suggestions tho. But I think if we put everything to env, it wouldn't be that obvious that our settings/constants file has these variables.